frontend/src/metabase-lib/v1/expressions/helper-text-strings.ts

```diff
@@ -285,7 +285,7 @@ const HELPER_TEXT_STRINGS: HelpTextConfig[] = [
     description: () => t`Returns the string of text in all lower case.`,
     args: [
       {
-        name: t`text`,
+        name: t`column`,
```

Comment:
In many cases we called this argument `column`, so I think it should be fine to do it here as well.

Reply:
This is a straightforward fix for the issue, but can we call the arguments `value` instead of `column` like we do in the `text` args? I feel that `column` might inadvertently suggest that it _has_ to be a column or field, and not a literal value.